Helper/Search/QueryBuilder.php

```diff
@@ -28,6 +28,7 @@ public function buildBody(): array
             'aggs' => $this->getAggs($hasPostFilter),
             'suggest' => $this->getSuggest(),
             'sort' => $this->getSort(),
+            'highlight' => $this->getHighlight()
```

Comment:
I think we can use $this->search->getHighlight() directly ?
The return null of getHighlight is not necessary, as array_filter considers [] as a value to remove.
@davma what is your opinion?

Reply:
Yes that would indeed be beter!